Formula/bitwarden-cli.rb

```diff
@@ -3,8 +3,8 @@
 class BitwardenCli < Formula
   desc "Secure and free password manager for all of your devices"
   homepage "https://bitwarden.com/"
-  url "https://registry.npmjs.org/@bitwarden/cli/-/cli-1.14.0.tgz"
-  sha256 "6f0d5e7c78afd7aa5c1b2f393638705700f2792101eebe57eac8bd30c94c9756"
+  url "https://registry.npmjs.org/@bitwarden/cli/-/cli-1.15.0.tgz"
+  sha256 "e9aa84a7b657356092bb811880d04f8a138ac4c0205d929a0d342d778136c058"
   license "GPL-3.0"
```

Comment:
```suggestion
  license "GPL-3.0-only"
```
There are basically no results for "at your option" https://github.com/bitwarden/cli/search?q=at+your+option

Reply:
I raised an issue for this, maybe we can delay the call for the next release.